### PR TITLE
Add azure-storage-* packages

### PR DESCRIPTION
**Pull request by xhochy**
These were previously a single azure-storage package but are now split
up into a package per service.

### CODE REVIEW

**Comment by xhochy** [comment]
This depends on https://github.com/conda-forge/azure-common-feedstock/pull/7

File: recipes/azure-storage-blob/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - azure
+    - azure.storage
+    - azure.storage.blob
+
+about:
+  home: https://github.com/Azure/azure-storage-python
+  license: Apache-2.0
```
Comment (CJ-Wright) [inline recipes/azure-storage-blob/meta.yaml]: This needs a license file, and a license family.

**Comment by xhochy** [comment]
The current release of these artefacts don't build anymore with the latest version of their dependencies due to 

```
  Complete output from command /Users/ukorn/miniconda3/conda-bld/azure-storage-blob_1524064689048/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/python -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-c9w767np-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpfsgswou6pip-wheel- --python-tag cp36:
  Wheel is not available, disabling bdist_wheel hook
  running bdist_wheel
  error: error in setup.cfg: command 'bdist_wheel' has no such option 'azure_namespace_package'
```

This is fixed upstream but not released. Waiting for the release to happen before I progress here.

File: recipes/azure-storage-blob/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipes/azure-storage-blob/meta.yaml]: Added both and raised an issue upstream to include the license file in the release tarballs: https://github.com/Azure/azure-storage-python/issues/477

**Comment by xhochy** [comment]
@conda-forge/staged-recipes 4 months later this should now be ready to go :)

**Pull request merged**